verifier/rest/rest.go

```diff
@@ -203,20 +183,67 @@ func convertRequestToREST(request verifier.VerifyAttestationRequest) *confidenti
 		},
 	}
 
-	if request.Attestation.GetSevSnpAttestation() != nil {
-		sevsnp, err := convertSEVSNPProtoToREST(request.Attestation.GetSevSnpAttestation())
-		if err != nil {
-			log.Fatalf("Failed to convert SEVSNP proto to API proto: %v", err)
+	if request.Attestation != nil {
```

Comment:
if you add the check that only one of Attestation or TDCCELAttestation are set above, these if statements are no longer necessary.

Reply:
removed the else if check

Reply:
still need to check the val is not nil, because the converter has a all empty protobuf check, which will fail if we don't check the nil